packages/lesswrong/components/review/NominatePostMenuItem.tsx

```diff
@@ -69,12 +77,13 @@ const NominatePostMenuItem = ({ post, closeMenu }: {
   
   return (<React.Fragment>
       <Tooltip title={tooltip} placement="left">
-        <MenuItem onClick={handleClick}>
-          <ListItemIcon>
+        {/* <MenuItem> */}
+          <ReviewVotingWidget post={post}/>
+          {/* <ListItemIcon>
             { nominated ? <StarIcon /> : <StarBorderIcon /> }
           </ListItemIcon>
-          {nominated ? "View Nomination" : "Nominate Post"}
-        </MenuItem>
+          {nominated ? "View Nomination" : "Nominate Post"} */}
+        {/* </MenuItem> */}
```

Comment:
My instinct is that even when rushing, we should clean this up before merging to devel.

Reply:
Actually I solved this the other way – I don't actually want to get rid of NominatePostMenuItem yet because we might still decide all this crazy newfangled Pre-Voting is a bad idea and want to revert to it. Instead just moved the new component elsewhere.